docs/Best Practices/Audio Books/index.md

```diff
@@ -2,9 +2,9 @@
 An audiobook app for SDL should include as much of the native apps functionality as possible, since that is what a user will be expecting out of the app.
 
 #### Use Voice Commands
-Your Audiobook app should contain such basic functionality as `Play`, `Pause`, and `Resume`. You can use build in controls for these features by calling the `subscribeButton` RPC. This will allow you to not only create buttons on the infotainment system but also be informed when the user presses hard buttons, which might be placed on the steering wheel. It is important to also add voice control capabilities for these functions. To add a voice command use the `addCommand` RPC but omit the `menuParams` argument. This will create a voice command without creating an item in the menu.   
+Your Audiobook app should contain such basic functionality as `Play`, `Pause`, and `Resume`. You can use build in controls for these features by calling the `subscribeButton` RPC. This will allow you to not only create buttons on the infotainment system but also be informed when the user presses hard buttons, which might be placed on the steering wheel. It is important to also add voice control capabilities for these functions. To add a voice command use the `AddCommand` RPC but omit the `menuParams` argument. This will create a voice command without creating an item in the menu.   
 
-##*Add link to AddCommand Section*
+More information about this RPC can be found on the [AddCommand page](https://smartdevicelink.com/en/docs/hmi/master/ui/addcommand/).
```

Comment:
```suggestion
More information about this RPC can be found on the [UI.AddCommand](https://smartdevicelink.com/en/docs/hmi/master/ui/addcommand/) page.
```

Reply:
updated in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/2d12ff2688a645cbbb6dcad8fe764e24ae77cb73, also capitalized PerformInteraction below.